smac/runhistory/runhistory.py

```diff
@@ -358,22 +348,26 @@ def load_json(self, fn: str, cs: ConfigurationSpace):
         cs : ConfigSpace
             instance of configuration space
         """
-
-        # lock `fn` (writing)
-        lock_file_name = fn + ".lock"
-        with filelock.FileLock(lock_file_name):
+        try:
             with open(fn) as fp:
                 all_data = json.load(fp, object_hook=StatusType.enum_hook)
-        try:
-            os.remove(lock_file_name)
-        except FileNotFoundError:
-            pass
+        except Exception as e:
+            self.logger.warning(
```

Comment:
I can't figure out whether this is initialized but I assume so :-)
Also, is somewhere a logging output that we successfully read in some data points?

Reply:
Good catch, of course it wasn't ;) There's some logger output in the pSMAC part of code - so you get the output if you're using pSMAC.